cpp/oneapi/dal/algo/triangle_counting/backend/cpu/intersection_tc.hpp

```diff
@@ -68,6 +71,34 @@ ONEDAL_FORCEINLINE std::int32_t _popcnt32_redef(const std::int32_t& x) {
     {}
 #endif
 
+#ifdef __ARM_ARCH
```

Comment:
This implementation looks the same as the default one above, so the specialisation isn't required at the moment.